web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -506,6 +506,13 @@ $.extend(fixmystreet.set_up, {
     });
     // Sort elements in place, but leave the first 'pick a category' option alone
     $group_select.find("option").slice(1).sort(function(a, b) {
+        // 'Other' should always be at the end.
+        if (a.label === 'Other') {
```

Comment:
Just to note translation issue here (probably could stick a class on the Other `<option>` when output, or add the string to translations, hmm), but think also probably okay to leave for the moment.